### DISCUSSION TITLE
Java Version out of Date

### DISCUSSION THREAD

**Discussion by sjdwhiting** [Security-Onion-Solutions/securityonion#4849] [Unsupported Versions] [answered]
Hello all,

I have an installation of SO that is showing out of date versions of Java running inside some of the containers (Logstash, elasticsearch). It looks like they are only a 1 or 2 versions behind. 

Is there a way to update these myself without breaking the install or do I need to wait for updated containers to be pushed out with SO updates?

**Comment by dougburks** [comment, marked as answer]
We use the standard Docker images from Elastic. Please wait for new images to be pushed out. Security Onion 2.3.70 will have Elastic 7.13.3 (or higher):
https://github.com/Security-Onion-Solutions/securityonion/issues/4730
